src/main/scala/evaluation/AugmentedExamplesEvaluator.scala

```diff
@@ -0,0 +1,69 @@
+package evaluation
+
+import scala.reflect.ClassTag
+import breeze.linalg._
+import breeze.numerics._
+
+import org.apache.spark.rdd.RDD
+import nodes.util.MaxClassifier
+
+object AugmentedExamplesEvaluator extends Serializable {
+
+  object AggregationPolicyType extends Enumeration {
+    type AggregationPolicyType = Value
+    val average, borda = Value
+  }
+
+  def averagePolicy(preds: Array[DenseVector[Double]]): DenseVector[Double] = {
+    preds.reduce(_ + _) :/ preds.size.toDouble
+  }
+
+  // * Let s(k) be the ordering of patch k.
+  // * Let s(k)[i] be an integer, so that the worst item is ranked 1 and the best item is ranked n.
+  // For i in images,
+  //  For k in patches,
+  //   score[i] += s(k)[i]
+  def bordaPolicy(preds: Array[DenseVector[Double]]): DenseVector[Double] = {
+    val ranks = preds.map { vec =>
+      val sortedPreds = vec.toArray.zipWithIndex.sortBy(_._1).map(_._2)
+      val rank = DenseVector(sortedPreds.zipWithIndex.sortBy(_._1).map(x => x._2.toDouble))
+      rank
+    }
+    ranks.reduceLeft(_ + _)
+  }
+
+  def apply[T: ClassTag](
```

Comment:
Given my comment above - I wonder if this names RDD is the right thing to do, or whether the grouping to a `Array[DenseVector[Double]]` makes sense in another node? What I'm struggling with a little bit conceptually here is the logical pipeline that we want to describe. Conceptually, featurization is like a map from an Image to a Matrix of features. Those features are concatenated and a model is fit. At test time there is no concatenation and we need an averaging policy. 

This is just one way to look at it, though - and I guess it is worth sketching out variants of the pipeline on a whiteboard or something.